Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.8
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
```

Comment:
Hvorfor gå så langt, kan jo bare være /app

Reply:
greit for meg 
@stigbd ?

Reply:
445e15d